packages/default/scss/upload/_layout.scss

```diff
@@ -157,201 +153,62 @@
-                height: 0;
-                border-width: 6px;
-                margin-top: -1px;
-                margin-right: -1px;
-            }
-
-            // Old icon rendering
-            .k-multiple-files-extension-wrapper::after,
-            .k-multiple-files-invalid-extension-wrapper::after {
-                top: -6px;
-                left: -6px;
-                width: 15px;
-                height: 35px;
-                border-width: 2px 0 0 2px;
-            }
-
-            // Old icon rendering
-            .k-file-extension,
-            .k-file-invalid-icon {
-                position: absolute;
-                bottom: 0;
-                line-height: normal;
-            }
-
-            // Old icon rendering
-            .k-file-invalid-icon {
-                margin-left: 4px;
-            }
-
-            .k-file-extension {
-                margin-left: .2em;
-                margin-bottom: .3em;
-                overflow: hidden;
-                text-overflow: ellipsis;
-                max-width: 100%;
-                left: 0;
-            }
-
-            .k-upload-action {
-                border-width: 0;
-                color: inherit;
-                background: none;
-                box-shadow: none;
-            }
         }
 
-        .k-upload-files .k-file-name-size-wrapper {
-            margin-left: 0;
-            margin-right: 0;
-            min-height: $upload-group-icon-size;
+        .k-upload-files .k-file-info {
+            min-height: $kendo-upload-group-icon-size;
             display: block;
             overflow: hidden;
             flex: 1 0 0;
         }
 
-        // old rendering
-        .k-file-multiple > .k-file-name-size-wrapper {
-            margin-left: $upload-item-padding-x;
-        }
-
         .k-multiple-files-wrapper,
-        .k-file-single > .k-file-name-size-wrapper,
-        .k-file > .k-file-name-size-wrapper {
+        .k-file-single > .k-file-info {
             margin-left: $padding-x;
-            margin-right: 0;
-            min-height: $upload-group-icon-size;
+            min-height: $kendo-upload-group-icon-size;
             display: block;
             overflow: hidden;
             flex: 1 0 0;
         }
 
         .k-multiple-files-wrapper {
-            .k-file-name-size-wrapper {
-                margin-bottom: $upload-multiple-items-spacing;
+            .k-file-info {
+                margin-bottom: $kendo-upload-multiple-items-spacing;
                 display: block;
             }
 
-            .k-file-information {
-                margin-top: -($upload-multiple-items-spacing / 3);
+            .k-file-summary {
+                margin-top: -($kendo-upload-multiple-items-spacing / 3);
             }
         }
 
-        // old rendering
-        .k-progress {
-            height: $upload-progress-thickness;
-            position: absolute;
-            bottom: 0;
-            left: 0;
-        }
-
         .k-progressbar {
```

Comment:
If you need to, sync with Emo about the progressbar. If needed add a css variable that sets the height of the progressbar.